_src/src/i18n.js

```diff
@@ -0,0 +1,55 @@
+const defaultLocale = "en";
+
+export default {
+    getLanguage() {
+        // Chrome and Firefox 32+
+        return navigator.languages || [
+          navigator.language ||
+          navigator.browserLanguage
+        ]
+    },
+    loadLocale(locale) {
+        return require(`./locale/${locale}`);
```

Comment:
What does this transpile to? I can think of two possible outcomes:
- It does HTTP request every time the function is called, or
- It simply doesn't work as intended

If it's any of the above, we need to change something. In the first case, testing *all* `navigator.languages` elements *twice* isn't a way to go. And if this doesn't work, well, it doesn't work.

Reply:
The webpack loader will build the Jason file into the finial JavaScript file, so it don't sent  HTTP request